bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -288,7 +288,9 @@ pub mod pallet {
 
 			// No fee for governance order
 			if !order.fee.is_zero() {
-				let reward_account_location = envelope.reward_address.into();
+				let account_bytes: [u8; 32] =
+					envelope.reward_address.encode().try_into().unwrap();
+				let reward_account_location = Location::new(0, AccountId32{id: account_bytes.into(), network: None});
```

Comment:
@claravanstaden I also made a PR for the compile error before I saw this one and deprecated it in favor of yours.  

But the `envelope.reward_address.encode()` here seems not correct, please check the fix in https://github.com/vgeddes/polkadot-sdk/pull/2 if that make sense.

Reply:
@yrong, great thanks! Will pull https://github.com/vgeddes/polkadot-sdk/pull/3 once merged. 😄

Reply:
@claravanstaden @vgeddes I've made https://github.com/vgeddes/polkadot-sdk/pull/3 ready.  

The main change in that PR is just one commit https://github.com/vgeddes/polkadot-sdk/pull/3/commits/148525ec4d4b264cb72ee663dfe9e8caaae99266 to use native ether as fee asset.

We may still need to revamp the integration/smoke tests accordingly but at least it compiles.

There is still https://github.com/paritytech/polkadot-sdk/pull/7402#issuecomment-2632581527 not finalized, not sure what you think of it. I'm fine with either option to add the `dry_run` back or not.

Reply:
Thanks @yrong, I have reviewed it - looks good.